### DISCUSSION TITLE
What is the use of go httpx?

### DISCUSSION THREAD

**Discussion by teo-en-ming** [projectdiscovery/httpx#475] [General] [answered]
Good day from Singapore,

May I know what is the use of go httpx?

Regards,

Mr. Turritopsis Dohrnii Teo En Ming
Targeted Individual in Singapore
3rd Jan 2022 Monday

**Comment by ehsandeep** [comment, marked as answer]
Please refer to https://github.com/projectdiscovery/httpx#readme
